src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -87,4 +90,10 @@ public void updateState(boolean reactiveLimits, boolean writeSlackBus, boolean d
     public boolean isParticipating() {
         return participating;
     }
+
+    @Override
+    public List<BusResult> createBusResults() {
+        return bus.getVoltageLevel().getBusBreakerView().getBusesFromBusViewBusId(bus.getId())
```

Comment:
Does it still work when if bus is a bus/breaker one?

Reply:
No :-) I am on it.